### ISSUE TITLE
Bring Crayons to file upload

### ISSUE THREAD

**Issue by Link2Twenty** [forem/forem#18151]
**Describe the request**
I noticed the button in the file uploaded is still the standard HTML button, it would be nice to crayonify this.

**To Reproduce**
1. go to [settings](https://dev.to/settings)
2. scroll down to user
3. see upload option

**Expected behaviour**
It would be nice to have a blue forem-esc button.

**Screenshots**
**Current**
![current upload button](https://user-images.githubusercontent.com/3534427/179615682-344330be-a68b-4dec-be45-da7df490b556.png)

**New**
![updated upload button](https://user-images.githubusercontent.com/3534427/179615747-c17b7385-d526-458c-861f-3b643d8adb33.png)

**Additional context**
I've included a [jsfiddle](https://jsfiddle.net/link2twenty/ysh2mcde/) mock up. Styles don't have to go this exact route but I thought I'd get the ball rolling.

**Comment by aitchiss** [comment]
Pinging @anujbhavsar96 in case he has any thoughts on this one 😄 

Definitely think it would be useful to create a design-system level file upload input. I'm not sure if it should maybe use secondary button variant though, since form submit would likely employ the primary one 🤔

**Comment by anujbhavsar96** [comment]
You are right @aitchiss , secondary button would be more appropriate here ✨

**Comment by Link2Twenty** [comment]
So more like this?

![revised image](https://user-images.githubusercontent.com/3534427/180164767-ab634b20-13c8-46ac-9fc9-64e1b42b586f.png)
